0-bootstrap/variables.tf

```diff
@@ -88,6 +88,12 @@ variable "bucket_tfstate_kms_force_destroy" {
   default     = false
 }
 
+variable "vpc" {
+  description = "The type of VPC to attach the project to. Possible options are base or restricted."
+  type        = string
+  default     = "null"
```

Comment:
Hi @mariammartins! Is there an advantage to use string `"null"` rather than `null`?

Reply:
Hi @apeabody, as we have two types of network "base" and "restricted", the proposal is to use the string "null" in all projects that do not have a network, this way, its easier to check if all projects are labeled consistently.
Using an empty string deviates from the label standard off putting the network type in the label of projects with vpc.

Reply:
Thanks @mariammartins! Just a suggestion, I personally might find `"none"` to be less confusing than `"null"` for no network.  Also might be safer given the overloaded meaning of `null`.

Reply:
@apeabody great suggestion, it makes more sense to use the value of none, so as not to cause confusion in the context of its use, change made.